uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/decorators/route.decorator.spec.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-env jasmine */
+
+(function () {
+  'use strict';
+
+  describe('RouteDecorator', function () {
+    var $route, crmApiSpy;
+
+    beforeEach(module('ngRoute', 'crm-tasks-workflows.decorators', function ($routeProvider) {
+      // This happens before the configuration phase:
```

Comment:
We also should leave a comment on why we have empty funcs in the code and what's happening to these funcs afterwards. Imho, so up to you.